src/Console/Command/RunCommand.php

```diff
@@ -409,9 +417,10 @@ protected function executionLoop(ProcessSet $processSet)
                         }
 
                         $testcaseFinishedMessage = sprintf(
-                            'Finished execution of testcase "%s" (result: %s)%s',
+                            'Finished execution of testcase "%s" (result: %s, time: %.1F sec)%s',
                             $testClass,
                             $processWrapper->getResult(),
+                            $testcaseEnd->getDuration() / 1024,
```

Comment:
🙈 1 second is not 1024 ms?

Nice catch, will fix this :). It occurred because I played with printing memory usage as well - but it was not much useful so I did not include it in the end.